Godeps/_workspace/src/gopkg.in/fatih/set.v0/set_ts.go

```diff
@@ -10,7 +10,7 @@ type Set struct {
 	l sync.RWMutex // we name it because we don't want to expose it
 }
 
-// New creates and initialize a new Set. It's accept a variable number of
+// New creates and initialize a new Set. It accepts a variable number of
```

Comment:
ok, reverted via git checkout 702218008ee2b6d708d6b2821cdef80736bb3224  Godeps/_workspace/src/gopkg.in/fatih/set.v0/set_ts.go on latest commit..tx!